packages/admin-web/src/components/AuthStepperPage.tsx

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React, { useState } from 'react';
+import TextField from '@material-ui/core/TextField';
+import { SimpleStepper, SimpleStepperStep } from '@backstage/core-components';
+import {
+  Button,
+  FormControlLabel,
+  Radio,
+  RadioGroup,
+  Typography,
+  List,
+  ListItem,
+} from '@material-ui/core';
+
+const AuthStepperPage = () => {
+  const [required, setRequired] = useState<boolean>(false);
```

Comment:
FWIW, there's a bit of potentially-confusing terminology overlap here.
Generally, the phrase `required` is a ["constant" attribute applied to an `input`](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/required) that defines whether it needs to be filled out or not. I'm calling the attribute "constant" as in: the attribute is always attached to the input, even when a value is added or removed (and the browser will properly mark it as invalid if `required` and there's no value).

Here, I think you're using the term `requires` as "is the current step 'valid' and it's OK to go to the next step". Perhaps the term `valid` (or, better yet, maybe `formIsValid`?) would be more appropriate?

Reply:
yeah right `formIsValid` sounds better that `require`